addons/website_slides/models/slide_channel.py

```diff
@@ -138,6 +139,10 @@ def _default_access_token(self):
         'mail.template', string='Share Template',
         help="Email template used when sharing a slide",
         default=lambda self: self.env['ir.model.data'].xmlid_to_res_id('website_slides.slide_template_shared'))
+    enroll_request_template_id = fields.Many2one(
```

Comment:
I would put an ondelete restrict on that field, so that people cannot remove them if used.